src/components/sdk/MeetingControls/ContentShareControl.tsx

```diff
@@ -16,12 +16,15 @@ interface Props {
   pauseLabel?: string;
   /** The label that will be shown for unpausing content share button in content share control, it defaults to `Unpause`. */
   unpauseLabel?: string;
+  /** Title attribute for the icon, it defaults to `Screen share`. */
+  title?: string;
```

Comment:
Would it be better to rename this prop to either `contentShareIconTitle` or `iconTitle`?

Reply:
Yep - I prefer iconTitle, I think - I'll make this change